srcpkgs/enyo-doom/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'enyo-doom'
 pkgname=enyo-doom
-version=1.06.9
+version=1.2
```

Comment:
This will not work. XBPS assumes (sensibly) that `06 > 2` and will not recognize this as an update.

The *right* thing to do is ask upstream to fix this nonsense sequencing and drop a 1.7 release or something. Given the author lists the project as dead, this may not be likely. We can work around it with

    reverts="1.04_1 1.05_1 1.06_1 1.06.9_1"

right before the `version` line. If you do this, please double-check the versions and revisions being reverted. I just took a quick look through the git logs without verifying the details.

Reply:
From a quick jaunt through git history, that seems correct.

Reply:
true, didnt even notice that my local package didnt get updated.